### ISSUE TITLE
Audio plays through default system speakers instead of Index HMD Audio

### ISSUE THREAD

**Issue by Dielji** [Raicuparta/nomai-vr#298]
When running the game with VR mod enabled, all audio plays out through the default system speakers instead of my Index HMD.  Changing the system audio output doesn't change the game's audio output once the game is running.  The Index HMD isn't available as an audio output until after SteamVR starts, so setting it as the default output before starting the game isn't an option.  I've even tried reversing SteamVR's audio mirroring setting so that the HMD is mirroring the default speakers, but the game seems to ignore those settings entirely and only play through whatever was the default audio device when it first started.

**Comment by Raicuparta** [comment]
Don't think there's much I can do about this. Did you try starting the game with SteamVR already open? I know the instructions say to do it differently, but some people are able to start it that way too (me included).

**Comment by Dielji** [comment]
That didn't work at first, but it led me to a solution.  Opening SteamVR first was making the game launch in a VR Theater Mode, so I had to go into the properties for the game on Steam and disable "Use Desktop Game Theatre while SteamVR is active."  After that I had to restart both the mod manager and SteamVR, then I was able to open SteamVR before launching the game, and everything worked as expected.

It might be worth making a note in the readme to disable Theater Mode and launch SteamVR first if people can't get audio to play correctly with the Steam version of the game.
